docs/docs/customization.md

```diff
@@ -56,3 +56,28 @@ For normal keyboards, follow the same flashing instructions as before to flash y
 
 For split keyboards, only the central (left) side will need to be reflashed if you are just updating your keymap.
 More troubleshooting information for split keyboards can be found [here](troubleshooting.md#split-keyboard-halves-unable-to-pair).
+
+## Building Additional Keyboards
+
+You can build additional keyboards with GitHub actions by appending them to `build.yml` in your `zmk-config` folder. The following example will build Corne with Nice!Nano and Lily58 with Nice!Nano v2:
```

Comment:
I think it can be made clearer that we are starting with Corne and adding a Lily58, e.g.
```suggestion
You can build additional keyboards with GitHub actions by appending them to `build.yml` in your `zmk-config` folder. For instance assume that we have set up a Corne shield with nice!nano during [initial setup](user-setup.md) and we want to add a Lily58 shield with nice!nano v2. The following is an example `build.yaml` file that would accomplish that:
```
I also changed the styling of nice!nano to be more consistent with the rest of the docs.